ocs_ci/utility/utils.py

```diff
@@ -518,9 +518,17 @@ def expose_ocp_version(version):
         str: Version of OCP exposed to full version if latest nighly passed
 
     """
+    ocp_arch = config.DEPLOYMENT['ocp_arch']
     if version.endswith(".nightly"):
+        if ocp_arch == "s390x":
+            path = "https://openshift-release-s390x.apps.ci.l2s4.p1.openshiftapps.com/"
```

Comment:
this URL should not be saved here..
@petr-balogh should it be loaded from a conf file?

Same for the other URLs

Reply:
@ebenahar @petr-balogh : Made this a configuration option.